### ISSUE TITLE
Implement Dark Mode for Zoo Verse Website

### ISSUE THREAD

**Issue by Jyothishree2k5** [ankitXD/ZMS#18]
Currently, the Zoo Verse website only supports light mode. To enhance user experience and accessibility, a dark mode option should be introduced. This will allow users to switch between light and dark themes according to their preference.

I would like to work on this issue ,can you assign it to me under Hacktoberfest?. Thank you!
